### DISCUSSION TITLE
Why Windows is not supported anymore?

### DISCUSSION THREAD

**Discussion by mrNo0b** [emqx/emqx#13322] [Q&A] [answered]
Since EMQX v5.4 Windows is not supported anymore, why?

**Comment by ieQu1** [comment, marked as answer]
https://github.com/emqx/emqx/discussions/13058#discussioncomment-9500519

- No one in our team uses Windows, so it's very hard for us to troubleshoot problems on that platform.
- Windows builds were never targeted for production use, only for evaluation.
- Windows users can use WSL.

**Comment by mrNo0b** [comment]
Can you release Windows versions but with note that its not supported, only for evaluation and provided without any warranty etc?

**Reply by ieQu1** [reply]
That was the state before deprecation. Maintaining the CI pipeline and fixing build issues took a lot of resources. Also, some features were unavailable on Windows anyway due to lack of support from 3rd party dependencies. That forced us to maintain different paths in the code and build configuration options.

**Reply by mrNo0b** [reply]
I understand, Thank you!
